<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotePad</class>
 <widget class="QMainWindow" name="NotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar
{
	font: 9pt &quot;微软雅黑&quot;;
}
QMenu
{
	font: 9pt &quot;微软雅黑&quot;;
}


</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Mod">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="separator"/>
    <addaction name="action_trim"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_del"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
    <addaction name="action_replace"/>
    <addaction name="separator"/>
    <addaction name="action_all"/>
    <addaction name="action_date"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Mod"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_trim"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
   <addaction name="separator"/>
   <addaction name="action_find"/>
   <addaction name="action_replace"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </widget>
  <action name="action_new">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/File new.png</normaloff>:/UI/icon/File new.png</iconset>
   </property>
   <property name="text">
    <string> 新建(N)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Save.png</normaloff>:/UI/icon/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Save as.png</normaloff>:/UI/icon/Save as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Bt left.png</normaloff>:/UI/icon/Bt left.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_trim">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Cut.png</normaloff>:/UI/icon/Cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Copy.png</normaloff>:/UI/icon/Copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Paste.png</normaloff>:/UI/icon/Paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_del">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Trash.png</normaloff>:/UI/icon/Trash.png</iconset>
   </property>
   <property name="text">
    <string>删除(L)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Search.png</normaloff>:/UI/icon/Search.png</iconset>
   </property>
   <property name="text">
    <string>查找(F)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_replace">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/replace.png</normaloff>:/UI/icon/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换(R)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_all">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/selectAll.png</normaloff>:/UI/icon/selectAll.png</iconset>
   </property>
   <property name="text">
    <string>全选(A)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_date">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Folder temporary.png</normaloff>:/UI/icon/Folder temporary.png</iconset>
   </property>
   <property name="text">
    <string>时间/日期(D)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/File import.png</normaloff>:/UI/icon/File import.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/File del.png</normaloff>:/UI/icon/File del.png</iconset>
   </property>
   <property name="text">
    <string>关闭(C)</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/exit.png</normaloff>:/UI/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/UI/icon/Help.png</normaloff>:/UI/icon/Help.png</iconset>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
   <property name="toolTip">
    <string>说明</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
